src/PersonObjects/Sleeve/Sleeve.ts

```diff
@@ -171,7 +172,7 @@ export class Sleeve extends Person implements SleevePerson {
     this.storedCycles += numCycles;
     if (this.storedCycles < CyclesPerSecond || !this.currentWork) return;
     const cyclesUsed = Math.min(this.storedCycles, 15);
-    this.shock = Math.max(0, this.shock - 0.0001 * cyclesUsed);
+    this.shock = this.shock - 0.0001 * calculateIntelligenceBonus(this.skills.intelligence, 0.75) * cyclesUsed;
```

Comment:
The `Math.max(0, value)` here was preventing shock from going negative, it's still needed.
![image](https://user-images.githubusercontent.com/84951833/220217031-3ed32974-e184-4a0c-a888-0778d2023f99.png)

Reply:
Thank you for pointing that one out, and my bad on that, much appreciated.